src/client/delphi_epidata.py

```diff
@@ -71,9 +78,19 @@ def _list(values):
     def _request_with_retry(endpoint, params={}):
         """Make request with a retry if an exception is thrown."""
         request_url = f"{Epidata.BASE_URL}/{endpoint}/"
+        if Epidata.debug:
+            Epidata.logger.info(f"Sending GET request to URL: {request_url} | params: {params} | headers: {_HEADERS}")
+        if Epidata.sandbox:
+            return
```

Comment:
how about returning a dummy `Response` object (since thats whats expected downstream)?  this one should be minimal but mostly valid json:
```suggestion
            resp = requests.Response()
            resp._content = b'true'
            return resp
```